docs/detections/api/exceptions/lists-index-api-overview.asciidoc

```diff
@@ -4,9 +4,10 @@
 Before using exceptions and lists, use the `index` endpoint to create `.lists`
 and `.items` system data streams in the relevant
 {kibana-ref}/xpack-spaces.html[{kib} space].
+//Need to follow up on this
```

Comment:
Do you still need this? Maybe create a separate ticket for any follow-up work.

Reply:
Whoops, don't need this anymore.

```suggestion
```